src/session/user.rs

```diff
@@ -146,8 +148,40 @@ mod imp {
 
             let avatar = obj.avatar();
             let user_expression = gtk::ConstantExpression::new(obj);
-            let full_name_expression = super::User::full_name_expression(&user_expression);
-            full_name_expression.bind(avatar, "display-name", gtk::NONE_WIDGET);
+            let display_name_expression = super::User::display_name_expression(&user_expression);
+            display_name_expression.bind(avatar, "display-name", gtk::NONE_WIDGET);
+        }
+    }
+    pub fn time_ago(unix: i32) -> String {
+        let current_time = Utc::now().timestamp();
```

Comment:
Is pulling in `chrono` as a new crate really neccessary?
I think the same can be archived with `glib::DateTime`, which is already there.
For example, start with
```rust
let now = glib::DateTime::new_now_utc().unwrap();
let unix = now.to_unix();
```
For reference, have a look at `src/session/sidebar/row.rs:215`.

Reply:
Thank you for your time and review. I also found more issues with time_ago() function. 
For now I'm converting this to draft. I'll come back when all errors will be resolved 🙂